lib/recipe/bloc/recipe_bloc.dart

```diff
@@ -314,6 +305,17 @@ class RecipeBloc extends Bloc<RecipeEvent, RecipeState> {
     emit(state.copyWith(recipes: recipes));
   }
 
+  bool _checkProbeAnswers(List<Probe> probes) {
+    bool allProbeAnswersStandard = true;
+    for (Probe probe in probes) {
+      if (probe.standardAnswer() == false) {
+        allProbeAnswersStandard = false;
+        break;
+      }
+    }
+    return allProbeAnswersStandard;
+  }
+
```

Comment:
Again, I was thinking that perhaps this could just be `_areProbeAnswersStandard()` or maybe `_areAllProbeAnswersStandard()` and perhaps could also be made a part of the RecipeState such that it could be called without any arguments.